test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,20 @@
 const { joinNames } = require("../src");
 
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  test("returns string of names, seperated by commas and an ampersand", () => {
+    expect(
+      joinNames([{ name: "Bart" }, { name: "Lisa" }, { name: "Maggie" }])
+    ).toBe("Bart, Lisa & Maggie");
+  });
+  test("returns string of names, seperated by commas and an ampersand", () => {
```

Comment:
Common convention is to leave a clear line between each of your tests to make them easier to read at a glance (also a typo in 'separated'):
```
  });

  test("returns string of names, separated by commas and an ampersand", () => {
```